examples/libcloudforensics.py

```diff
@@ -50,7 +50,7 @@ def AddParser(provider, provider_parser, func, func_helper, args=None):
     func (str): The name of the function to look for in the given provider
         and to add parsing options for.
     func_helper (str): A helper text describing what the function does.
-    args (list(tuple(str, str, str|None))): Optional. A list of arguments to add
+    args (list[tuple[str, str, str|None]]): Optional. A list of arguments to add
```

Comment:
Are you saying that it can be a list of tuples (e.g. `[(a, b, c)]`) *OR* a list of `Nones` (e.g. `[None]`)? If so it might cause problems if we're trying to unpack on iteration:

```python
for action, helper, default in args:
  # do something
```

This will fail if the list contains a non-tuple element.

For more complex structures like this, consider using [NamedTuples](https://docs.python.org/3/library/collections.html#collections.namedtuple)

Reply:
I'm trying to say that this is a list of tuples in which each tuple's 3rd element might be a `str` or `None`